scripts/gather.ts

```diff
@@ -103,6 +103,8 @@ class Gatherer {
   }
 
   collectLink(a: HTMLAnchorElement) {
+    const pageTitle = document.title;
+    console.log(`Page title: ${pageTitle}`);
```

Comment:
This debug code should probably be removed.

Reply:
Yup, fixed! Forgot I'd left it in there.